cli/src/dev-server/plane.ts

```diff
@@ -110,8 +106,6 @@ export class LocalPlane {
       logsProcess.on('error', err => {
         if (err.message.includes('ENOENT')) {
           reject(new Error('Docker command not found. Make sure Docker is installed and in your PATH.'))
-        } else {
-          reject(new Error(`${err.toString()}\n${errorLogs.join('\n')}`))
```

Comment:
Here, we should still throw an error, but we just won't have the errorLogs included in it.

I think just this will do:
```
reject(err)
```

Reply:
got it